mediapipe/tasks/ios/audio/audio_classifier/utils/sources/MPPAudioClassifierResult+Helpers.mm

```diff
@@ -33,14 +33,21 @@ + (nullable MPPAudioClassifierResult *)audioClassifierResultWithClassificationsP
   NSInteger timestampInMilliseconds =
       (NSInteger)(packet.Timestamp().Value() / kMicrosecondsPerMillisecond);
 
-  if (!packet.ValidateAsType<std::vector<ClassificationResultProto>>().ok()) {
+  std::vector<ClassificationResultProto> cppClassificationResults;
+  if (packet.ValidateAsType<ClassificationResultProto>().ok()) {
+    // If `runningMode = .audioStream`, only a single `ClassificationResult` will be returned in the
```

Comment:
Should this be .audioClip?

Reply:
No it's .audioStream. In stream mode, a single classification result is returned per callback. If the passed in chunk for a timestamp is longer than the model input dimension, repeated callbacks are invoked for the same timestamp.  

In audio clips mode, the input clip is split internally if it exceeds the model dimension before inference and results for all the chunks are returned in an array.